internal/connector/connector.go

```diff
@@ -262,18 +261,14 @@ func runKubernetesConnector(ctx context.Context, options Options) error {
 		MinVersion: tls.VersionTLS12,
 	}
 
-	httpErrorLog := log.New(logging.NewFilteredHTTPLogger(), "", 0)
-
 	proxy := httputil.NewSingleHostReverseProxy(kubeAPIAddr)
 	proxy.Transport = proxyTransport
-	proxy.ErrorLog = httpErrorLog
 
 	metricsServer := &http.Server{
 		ReadHeaderTimeout: 30 * time.Second,
 		ReadTimeout:       60 * time.Second,
 		Addr:              options.Addr.Metrics,
 		Handler:           metrics.NewHandler(promRegistry),
-		ErrorLog:          httpErrorLog,
```

Comment:
I think we probably still want these `ErrorLog` set on all the `http.Server` so that the logs are sent to the same place with the same format. Since our logs generally output as JSON, I would expect zerolog to format these as JSON messages as well.  By removing the logger here they definitely won't be JSON.

Reply:
Thanks! I was looking for this